src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -2305,23 +2305,24 @@ class of curves.  If the j-invariant is not unique in the isogeny
              'y^2 + x*y + y = x^3 + (-36)*x + (-70)',
              'y^2 + x*y + y = x^3 + (130*e-356)*x + (2000*e-2038)',
              'y^2 + x*y + y = x^3 + 4*x + (-6)']
-            sage: G2.edges(sort=True)
-            [('y^2 + x*y + y = x^3 + (-130*e-356)*x + (-2000*e-2038)',
-              'y^2 + x*y + y = x^3 + (-36)*x + (-70)', None),
+            sage: G2.edges(sort=True,sort_vertices=True)
+            [('y^2 + x*y + y = x^3 + (-36)*x + (-70)',
+              'y^2 + x*y + y = x^3 + (-130*e-356)*x + (-2000*e-2038)', None),
              ('y^2 + x*y + y = x^3 + (-36)*x + (-70)',
               'y^2 + x*y + y = x^3 + (130*e-356)*x + (2000*e-2038)', None),
-             ('y^2 + x*y + y = x^3 + (-36)*x + (-70)',
-              'y^2 + x*y + y = x^3 + 4*x + (-6)', None)]
+             ('y^2 + x*y + y = x^3 + 4*x + (-6)',
+              'y^2 + x*y + y = x^3 + (-36)*x + (-70)', None)]
+
```

Comment:
remove this empty line to avoid warnings about missing `# needs sage.graphs ....`

Reply:
done in e22d5f1f335d23fcdc5e6d7df40d35750339634e